e2e-tests/constants/eligibility.js

```diff
@@ -1,4 +1,5 @@
 export const ELIGIBILITY = {
   MAX_COVER_AMOUNT_IN_GBP: 500000,
   MAX_COVER_PERIOD_YEARS: 2,
+  COVER_EXCEED_AMOUNT: 250000,
```

Comment:
Can we have this as part of `TOTAL_CONTRACT_VALUE` constants? Thinking something like this. What do you think?

```
LESS_THAN_250k: {...},
MORE_THAN_250k: {...},
AMOUNT_250K: 250000
```